scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -211,6 +225,35 @@ namespace <%= namespace %> {
             });
         }
 
+        public void SetFinalCheckoutFields(string email, ShippingFields? shippingFields, CompletionCallback callback) {
```

Comment:
I'm a little bit worried about the amount of public methods we're adding on `Cart`. Like an end user would probably not use this method but `Cart` by design is something meant for the end user to interactive with.